app/src/main/res/layout/main_fragment.xml

```diff
@@ -14,72 +14,67 @@
             android:layout_height="wrap_content"
 
             android:layout_marginTop="41dp"
-            android:hint="Select a recipe"
+            android:gravity="center_horizontal"
 
+            android:hint="@string/selectaRecipe"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            android:text="Select a Recipe"
-            tools:layout_editor_absoluteX="0dp"
-            android:gravity="center_horizontal" />
+            app:layout_constraintTop_toTopOf="parent" />
```

Comment:
Re-designed the layout and solved the errors.
Source:
https://www.youtube.com/watch?v=vXjJyLpydxg